ROMFS/px4fmu_common/mixers-sitl/autogyro_sitl.main.mix

```diff
@@ -0,0 +1,37 @@
+Mixer for SITL plane
```

Comment:
I think this is not a plane, or if it is identical with the plane can we use that instead?

Reply:
Yes, it is not same as plane mixer, fixed 9122b40a875762f5903cb142b73391ddb48791ac